framework/branch-key-store.md

```diff
@@ -31,12 +35,22 @@ The Keystore persists branch keys in DynamoDb that wrap multiple data keys.
 This creates a hierarchy where a branch key wraps multiple data keys and facilitates caching.
 These branch keys are only generated using the [AWS KMS API GenerateDataKeyWithoutPlaintext](https://docs.aws.amazon.com/kms/latest/APIReference/API_GenerateDataKeyWithoutPlaintext.html).
 
+By creating and persisting a data key to an accesbile medium,
+such as a DynamoDb table,
+distributed cryptographic agents can use a common, coordianted, data key
+as the root of a key hierarchy.
+
+This prevents distributed cryptographic agents from independently
+generating unique data keys that COULD BE coordinated,
+which leads to poor caching performance at decryption,
```

Comment:
```suggestion
which leads to poor decryption performance,
```

Reply:
What is "poor decryption performance"?

How does one "poorly" decrypt a message?

Do they fail to check the Auth Tag?

Do they do it in a particularly slow manner?

I am detailing the aspect of the Caching CMM that behaves poorly at Decryption,
which is not Decryption,
the CachingCMM does the same Decryption as any of the other CMMs or Keyrings.

The particular aspect of the CachingCMM that is second rate in Decryption
is Caching Performance:
the probability of a cache hit is inversly proportional to the number of hosts that encrypted data.
i.e: More Encrypting Hosts, worse probability of a Cache hit.

That is not "poor decryption performance".
The Performance of Decryption is, fortunately, static.

That is poor Caching performance.